source/SAMRAI/mesh/GriddingAlgorithm.C

```diff
@@ -3047,13 +3081,21 @@ GriddingAlgorithm::fillTagsFromBoxLevel(
          if (!preserve_existing_tags) {
             tag_data->fill(tag_value, box);
          } else {
+#if defined(HAVE_RAJA)
+            auto tags = tag_data->getView<2>();
+            auto false_tag = d_false_tag;
+            pdat::parallel_for_all(box, [=] SAMRAI_HOST_DEVICE (int k, int j) {
```

Comment:
Now that we are allocating tag data on the host, these changes could be backed out. The only reason to keep them would be to do these copies in parallel on the host using OpenMP. Any thoughts @nselliott ?

Reply:
Our target right now is cuda-based acceleartion on the GPU, so it makes sense to remove this.